core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -221,6 +222,184 @@ private[spark] class BlockManager(
     new BlockManager.RemoteBlockDownloadFileManager(this)
   private val maxRemoteBlockToMem = conf.get(config.MAX_REMOTE_BLOCK_SIZE_FETCH_TO_MEM)
 
+  /**
+   * @param blockSize the decrypted size of the block
+   */
+  private abstract class BlockStoreUpdater[T](
```

Comment:
I'd add a very short description of the class to this doc, eg. "Abstraction for storing blocks from bytes, whether they start in memory or on disk."